src/client/src/components/pages/participants/prisonDetails/staticData.js

```diff
@@ -0,0 +1,106 @@
+export const state = {
+    utrNo: "",
+    prison: "",
+    prisonNo: "",
+    username: "",
+    prisonIn: "",
+    prisonRelease: "",
+    rotlHdc: "",
+    offence: "",
+    prisonPointOfContact: ""
+  };
+  export const fields = [
+      [{
+              tag: "Input",
+              label: "UTR. No",
+              name: "utrNo",
+              width: "15rem",
+              placeholder: "UTR. No"
+          },
+          {
+              tag: "Input",
+              label: "Prison",
+              name: "prison",
+              width: "15rem",
+              placeholder: "prison"
+  
+          },
+          {
+            tag: "Input",
+            label: "Prison No",
+            name: "prisonNo",
+            width: "15rem",
+            placeholder: "prison No"
+    
+        }
+      ],
+      [{
+          tag: "Input",
+          label: "User Name",
+          name: "username",
+          width: "35rem",
+          placeholder: "username"
+  
+      }],
+      
+      [{
+              tag: "Input",
+              label: "Prison In",
+              name: "prisonIn",
+              width: "15rem",
+              type: "date",
+  
+          },
+          {
+              tag: "Input",
+              label: "Prison Release",
+              name: "prisonRelease",
+              width: "15rem",
+              type: "date",
+          },
+          {
+              tag: "Input",
+              label: "ROTL/HDC",
+              name: "rotlHdc",
+              width: "15rem",
+              placeholder: "ROTL/HDC"
+          }
+      ],
+      [{
+          tag: "Input",
+          label: "Offence",
+          name: "offence",
+          width: "20rem",
+          placeholder: "offence"
+  
+      },
+      {
+        tag: "Input",
+        label: "Prison Point Of Contact",
+        name: "prisonPointOfContact",
+        width: "20rem",
+        placeholder: "prison point of contact"
+
+    }],
+      [{
+          tag: 'Button',
+          value: "Add Prison Details",
+          color: "#272727",
+      }, {
```

Comment:
You need to add `staticField: true` to make the button appear in your form,

Reply:
@ishak52

Reply:
It's didn't when I run it locally with the last master branch version.
According to the last `form` component version, you should give `staticField` a boolean value (true or false) otherwise the button will not appear. @AsalaKM

Reply:
I got it . Many Thanks @ishak52

Reply:
:+1: